lib/request.coffee

```diff
@@ -45,12 +46,26 @@ prepareOptions = (options = {}) ->
 		gzip: true
 		json: true
 		headers: {}
+		refreshToken: true
 
 	options.url = url.resolve(settings.get('remoteUrl'), options.url)
 
-	utils.getAuthorizationHeader().then (authorizationHeader) ->
-		options.headers.Authorization = authorizationHeader
-		return options
+	Promise.try ->
+		return if not options.refreshToken
+
+		utils.shouldUpdateToken().then (shouldUpdateToken) ->
+			return if not shouldUpdateToken
+
+			exports.send
+				url: '/whoami'
+				refreshToken: false
+			.get('body').then (newToken) ->
```

Comment:
True. I moved to another line to add a `console.log` for debugging purposes and forgot to collapse it back.